src/board.py

```diff
@@ -1,12 +1,49 @@
+import string
+
 def construct_board():
-    return [[0] * 6] * 6
+    return [[0]*6 for _ in range(6)]
+
+def is_board_full(board):
+    for x, line in enumerate(board, 0):
+        for y, value in enumerate(line, 0):
+            if value == 0:
+                return False
+    return True
+
+def get_position_if_valid(board, position):  
```

Comment:
I don't understand the code of this method. Perhaps you could detail the input format and add comments?